src/main/java/no/fint/event/model/EventResponse.java

```diff
@@ -33,4 +39,12 @@ public ResponseStatus getResponseStatus() {
     public void setResponseStatus(ResponseStatus responseStatus) {
         this.responseStatus = responseStatus;
     }
+
+    public String getStatusCode() {
```

Comment:
Use @Getter / @Setter or @Data for this class?
Lombok is already used here anyway.